R/shiny.R

```diff
@@ -1269,7 +1269,9 @@ ShinySession <- R6Class(
         ext <- tools::file_ext(filename)
         if (nzchar(ext))
           ext <- paste(".", ext, sep = "")
-        tmpdata <- tempfile(fileext = ext)
+        tmpdata <- tempfile( tmpdir = tempdir(), fileext = ext)
+        snapshot <- utils::fileSnapshot(tempdir())
```

Comment:
It looks like his function was added in R 3.1.0 (see the history of https://github.com/SurajGupta/r-source/blob/db0ce24/src/library/utils/R/changedFiles.R#L19). I think that's probably OK, but we should also rev the required R version in the DESCRIPTION file.